rmf_fleet_adapter_python/scripts/rmf_fleet_adapter_python/test_adapter.py

```diff
@@ -405,22 +498,24 @@ def main():
     assert test_graph.get_waypoint(2).holding_point
     assert test_graph.get_waypoint(9).holding_point
     assert not test_graph.get_waypoint(10).holding_point
-
-    """
-                     10(D)
-                      |
-                      |
-                      8------9
-                      |      |
-                      |      |
-        3------4------5------6------7(D)
-                      |      |
-                      |      |
-                      1------2
-                      |
-                      |
-                      0
-   """
+    
+    test_graph_vis = \
+        """
+                         10(D)
```

Comment:
We can change the "D" here to "Ingestor" :sweat_smile:

Reply:
fixed!